apps/web/src/pages/integrations/constants.ts

```diff
@@ -5,6 +5,7 @@ export enum IntegrationsStoreModalAnalytics {
   UPDATE_INTEGRATION_FORM_SUBMIT = 'Update Integration Form Submit - [Integrations Store Modal]',
   SELECT_PROVIDER_CLICK = 'Select Provider Click - [Integrations Store Modal]',
   CLOSE_MODAL = 'Close Modal - [Integrations Store Modal]',
+  CREATE_INTEGRATION_INSTANCE = 'Create Integration Instance - [Integrations Multi Provider]',
```

Comment:
Not sure what naming would be best here

Reply:
I wonder if we can use the API event instead.